### PR TITLE
Add info to sizes dropdown

### PR DESCRIPTION
**Pull request by manzwebdesigns**
This PR includes the monthly cost and amount of RAM in the dropdown for the sizes menu.

### CODE REVIEW

File: digitaloceanmodule.php [hunk 1]
```diff
@@ -430,7 +430,7 @@ public function getSizesDropdown($module_row)
         $dp = array();
         foreach ($result->sizes as $key => $value) {
             if ($result->sizes[$key]->available == true) {
-                $dp[$result->sizes[$key]->slug] = $result->sizes[$key]->slug;
+                $dp[$tmp->slug] = $tmp->slug . '  ( $' . $tmp->price_monthly . ', RAM: ' . $tmp->memory . ')';
```
Comment (JReissmueller) [inline digitaloceanmodule.php:434]: Seems to me that $tmp is undefined, no?

**Review by JReissmueller — CHANGES_REQUESTED** [review]

File: digitaloceanmodule.php (hunk 1 above)
Comment (manzwebdesigns) [reply digitaloceanmodule.php:434]: Not sure how that happened, but I made the change in my repo.  Do I need to do another PR?
`$tmp = $result->sizes[$key];`

Comment (JReissmueller) [reply digitaloceanmodule.php:434]: Nope, looks good to me

Comment (manzwebdesigns) [reply digitaloceanmodule.php:434]: Aweome, thanks!

**Pull request merged**